### PR TITLE
fix(android): Recurrence rules for events fetched with getEventById are missing.

### PR DESCRIPTION
**Pull request by ypbnv**
**JIRA:** https://jira.appcelerator.org/browse/TIMOB-26799

**Description:**
Whenever an event is fetched through a getEventById call its recurrence rule (if there is one) can
not be accessed from the JS context.
Docs changes rely on having https://github.com/appcelerator/titanium_mobile/pull/10675 merged before this one. Still there may be conflicts so I will keep an eye on this one.

Note: This fix does not have unit tests for now, because it requires an accessible calendar. And it requires a Gmail account to be logged in. I will look for a solution for this issue.

**Test case:**
The JIRA ticket has a code snippet attached. It will require a calendar with an event with a recurring rule in it. Also the Calendar permissions to be added in `tiapp.xml`.

### CODE REVIEW

**Review by jquick-axway — APPROVED** [review]
CR: Pass

**Comment by ssekhri** [comment]
FR Passed. The details of the events with recurrence rule are fetched successfully.
Environment:
Studio: 5.1.2.201812191831
SDK: 8.1.0 local build
Mac OS: 10.14.3
Appc NPM: 4.2.13
Appc CLI: 7.0.10-master.17
Node: 8.12.0
NPM Ver: 6.4.1
Java Ver: 9.0.4

**Comment by ssekhri** [comment]
Waiting on Jenkins build before merge. Currently failing on android unit tests

**Pull request merged**
